open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -61,47 +61,45 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        if len(df[df["Fare"].isnull()]) &gt; 0:
</pre><p>create a function as clean_fare() for line38-41</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done